Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -542,6 +542,13 @@ script:
     - description: The tag for the new address.
       isArray: true
       name: tag
+    - auto: PREDEFINED
+      description: Whether to create the tag if it does not exist.
+      defaultValue: 'false'
+      name: create_tag
+      predefined:
+      - 'true'
+      - 'false'
```

Comment:
@MosheEichler didn't we say we shouldn't do it?

Reply:
@GuyAfik Added according @idovandijk's request.

Reply:
@GuyAfik I'm open to suggestions but the goal was to keep previous behavior while allowing the same behavior as seen in the UI, where if the user adds a tag it will get automatically created if it doesn't already exist

Reply:
@idovandijk i think its a bit strange to create tags in the `pan-os-create-address`, however to align with the UI makes sense.

my only concern is that customers would abuse it to create tons of tags which would actually make an api request per each tag, something that we want to avoid.